src/bosh_azure_cpi/lib/cloud/azure/azure_client2.rb

```diff
@@ -2417,7 +2417,10 @@ def http_post(url, body = nil, params = {}, retry_after = 5)
     end
 
     def merge_http_common_headers(request)
-      request['User-Agent'] = USER_AGENT_FOR_REST
+      user_agents = ["#{USER_AGENT_FOR_REST}/#{Bosh::AzureCloud::VERSION}"]
+      isv_tracking_guid = @azure_properties['isv_tracking_guid'].nil? ? DEFAULT_ISV_TRACKING_GUID : @azure_properties['isv_tracking_guid']
+      user_agents.push("pid-#{isv_tracking_guid}")
```

Comment:
The format has to be "pid-#{isv_tracking_guid}"? Or defined by CPI?

Reply:
has to be 'pid-xxx'

Reply:
Then it's better to add a comment to link to the doc.

Reply:
The link is only for internal currently. Can't add the link.